src/_articles/libraries/code/stream_controller_bad.dart

```diff
@@ -1,62 +0,0 @@
-// ignore_for_file: cancel_subscriptions
-import 'dart:async';
-
-// #docregion flawed-stream
-// NOTE: This implementation is FLAWED!
```

Comment:
Don't we still need a bad stream controller for documentation purposes?

Reply:
This file was used to link to a file from the article, but it is just a duplicate of the other `stream_controller_bad` file which can cause issues for keeping them in sync.

Like some of our other, newer documentation, instead of linking directly to the file(which opens the raw file in the browser), I've modified this to link to the original file in GitHub.